### DISCUSSION TITLE
One click DigitalOcean installer error

### DISCUSSION THREAD

**Discussion by SebBorbely** [inventree/InvenTree#4701] [Q&A] [answered]
Hi,

I am trying to install Inventree using the one click DigitalOcean installer. I have previously done this when the Inventree version number was 0.6.3 and that worked as expected. When trying to set up the currently supplied one with version number 0.9.2 I go through all the same setup procedures to create the DO droplet. Once the droplet is complete I try to open the droplet console to login as root user so that I can get the admin login details. Every time I try to open this using the DO console it start connecting, then immediately disconnects and I see this message.
![Capture](https://user-images.githubusercontent.com/99136411/234514580-dd218772-90ac-43a0-aa20-fa5a69a1b514.PNG)
I also try accessing it using command prompt and logging in through the ssh connection. I enter my password, it load for a second then exits as seen here.
![Capture1](https://user-images.githubusercontent.com/99136411/234515336-236f18ea-face-43d4-8ffa-32a4a522fbaa.PNG)

I have checked that the instance is running as you can see here.
![Capture2](https://user-images.githubusercontent.com/99136411/234515994-078398ca-a1dd-44f5-aa6c-13868c6889a2.PNG)

So I am wondering if I am doing something wrong or if there is an issue with the one-click installer?
If there is any useful information I have forgotten to mention please ask.
I hope someone can help me with this issue.

**Comment by matmair** [comment, marked as answer]
There was an error with the sshd setup in that version; a new image was supplied to Digitalocean and is being reviewed right now.
My suggestion would be to delete the droplet and wait till Digitalocean has checked the image. You could also use the recovery console to log in, uninstall openssh-server and install it while overwriting the settings.

**Reply by SebBorbely** [reply]
Thank you very much for looking into this and figuring out the issue.
I will wait for Digitalocean to review the image and then I will install the new version fresh. Any idea how long it could take for that to go through? (No worries if not)
I'll mark this as the answer and wait for the new image to go through before closing it if that is ok. Thanks

**Reply by matmair** [reply]
Sorry to say: I have no idea. As far as I can tell it is a manual process so it is up to luck. I can ping you if I notice it being up again.

**Reply by SebBorbely** [reply]
No worries. Thanks for the quick reply. Yeh if you notice it please do ping me. I shall also keep an eye out for it and post here to confirm that it is up and running and then close the discussion.

**Comment by SebBorbely** [comment]
The DO image is now up and running and I have successfully created a droplet with it. Thank you for your help in this matter and also for bringing the one-click install version up to date.
I will now close the discussion.
